File: src/TellDontAsk/Domain/Category.cs
using System;

namespace TellDontAsk.Domain
{
    public class Category
    {
        private String name;
        private decimal taxPercentage;

        public String getName()
        {
            return name;
        }

        public void setName(String name)
        {
            this.name = name;
        }

        public decimal getTaxPercentage()
        {
            return taxPercentage;
        }

        public void setTaxPercentage(decimal taxPercentage)
        {
            this.taxPercentage = taxPercentage;
        }
    }

}

File: typescript/src/domain/Category.ts
class Category {
  private name: string;
  private taxPercentage: number;

  public getName(): string {
      return this.name;
  }

  public setName(name: string): void {
      this.name = name;
  }

  public getTaxPercentage(): number {
      return this.taxPercentage;
  }

  public setTaxPercentage(taxPercentage: number) {
      this.taxPercentage = taxPercentage;
  }
}

export default Category;

